tox.ini

```
@@ -26,6 +26,10 @@ extras =
     tests
     coverage
 deps =
+  scipy
+  nltk
+  pandas
```

I don't think we need these dependencies, right?
```suggestion
  django-rosetta
```